models/comments.models.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.insertCommentsByArticleID = (article_id) => {
```

Comment:
I would watch your naming here, insert is reserved for a post request, maybe use fetchCommentsByArticleID :)